pgjdbc/src/test/java/org/postgresql/jdbc/InsertRewriteWithAlternatingTypesIssue584.java

```diff
@@ -59,7 +59,7 @@ public void tearDown() throws SQLException {
   }
 
   @Override
-  protected void updateProperties(Properties props) {
+  protected void updateProperties(PGProperties props) {
```

Comment:
in the tests it is, yes, the basetest does:

  protected void updateProperties(PGProperties props) {
    if (binaryMode == BinaryMode.FORCE) {
      forceBinary(props);
    }
  }

  protected void forceBinary(PGProperties props) {
    PGProperty.PREPARE_THRESHOLD.set(props, -1);
  }